lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -99,8 +99,8 @@
             <span ng-bind="flash.getProgressButtonLabel()"></span>
         </progress-button>
 
-        <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed && main.state.getFlashState().percentage != 100">
-          <span ng-bind="main.state.getFlashState().speed.toFixed(2) + ' MB/s'"></span>
+        <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed != undefined && main.state.getFlashState().percentage != 100">
```

Comment:
The speed should still display at 100% (because of compressed images etc)

Reply:
Done